apps/src/blockly/addons/shortcutMenuStyles.scss

```diff
@@ -0,0 +1,25 @@
+@use '@code-dot-org/component-library-styles/typography.module.scss' as
+  typography;
```

Comment:
Just curious: do we have a linter that wraps this early?

Reply:
Oh, I was curious why that line wrapped so early.  I thought there might be a SCSS linter that did it.

If not, I don't think it needs to wrap early, given that the same file has a longer line in it [here](https://github.com/code-dot-org/code-dot-org/blob/d5ded4cc2742d34272cb4ff6fdb295eef591f1ae/apps/src/templates/rubrics/rubrics.module.scss#L551).

Reply:
Ohhh I'm following! Yes it was a linter; it must wrap at 80. Not sure about that other file- I wonder if there are different rules for comments, or if they just didn't have format on save.